runtime/src/main/java/org/corfudb/util/serializer/CorfuSerializer.java

```diff
@@ -12,7 +12,7 @@
     private final byte type;
 
     /* The magic that denotes this is a corfu payload */
-    final byte corfuPayloadMagic = 0x42;
+    public static final byte corfuPayloadMagic = 0x42;
```

Comment:
why public?

Reply:
Good point. I'll just add a seek method to ISerializer and change this to private. We shouldn't expose this.